app/components/find/courses/summary_component/view.rb

```diff
@@ -50,6 +50,22 @@ def visa_sponsorship_row
             'Visas cannot be sponsored'
           end
         end
+
+        def course_fee_value
+          safe_join([formatted_uk_eu_fee_label, tag.br, formatted_international_fee_label])
+        end
+
+        def formatted_uk_eu_fee_label
+          return if course.fee_uk_eu.blank?
```

Comment:
It's not currently possible to publish a course without this, but didn't think there was any harm in adding it. 

<img width="525" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/740c7d83-db8e-42d9-8508-17e881b99914">